crates/uplc/src/ast.rs

```diff
@@ -92,6 +92,16 @@ impl Program<Name> {
 
         program
     }
+
+    /// A convenient method to convery named programs to debruijn programs.
+    pub fn to_debruijn(self) -> Result<Program<DeBruijn>, debruijn::Error> {
+        self.try_into()
+    }
+
+    /// A convenient method to convery named programs to named debruijn programs.
+    pub fn to_named_debruijn(self) -> Result<Program<NamedDeBruijn>, debruijn::Error> {
+        self.try_into()
+    }
```

Comment:
Not sure why I didn't do that sooner. Been so annoyed with this when implementing the prop test framework. 

e.g. https://github.com/aiken-lang/aiken/blob/fdbe811bf1a547de0dfe2e66a2912d5d2985accf/crates/aiken-project/src/test_framework.rs#L62

Reply:
yea I had enough of that and made a helper lol